lib/ui/scripts/config.js

```diff
@@ -0,0 +1,55 @@
+import path from 'path';
+import webpack from 'webpack';
+
+const resolveLocal = dir => path.join(__dirname, dir);
+
+const r = resolveLocal('../../../node_modules');
+
+const out = resolveLocal('../../core/dll');
+export default ({ entry, provided = [] }) => ({
+  name: 'storybook-ui',
+  mode: 'production',
+
+  entry,
+  output: {
+    path: out,
+    filename: '[name]_dll.js',
+    library: '[name]_dll',
+  },
+  externals: provided,
+
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        use: [
+          {
+            loader: 'babel-loader',
+            options: {
+              presets: [
+                ['@babel/preset-env', { shippedProposals: true, modules: false }],
```

Comment:
I feel like i've seen this above. Maybe the babel rcs could be moved to a common location?

Reply:
Sounds like an improvement/refactor we could do a bit later. Good suggestion!